circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  environment:
+    RABBIT_VERSION: 3.6.5-management
+  services:
+    - docker
+
+dependencies:
+  cache_directories:
+    - "~/docker"
+  override:
+    - docker info
+    - if [[ -e ~/docker/image.tar ]]; then docker load --input ~/docker/image.tar; fi
+    - docker build -t unifio/rabbitmq .
+    - mkdir -p ~/docker; docker save unifio/rabbitmq > ~/docker/image.tar
+
+test:
+  override:
+    - docker run -d --hostname my-rabbit --name some-rabbit unifio/rabbitmq
+    - docker ps | grep rabbitmq
```

Comment:
What if the container is in the process of rebooting? Sometimes you may not know that the container is in a stable state until it has been running for at least a few seconds.  Is this a simple sanity check?